kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/LinuxNetworkUtil.java

```diff
@@ -264,22 +269,45 @@ private boolean isWifiLinkUpInternal(String ifaceName) throws KuraException {
     }
 
     public static boolean toolExists(String tool) {
-        boolean ret = false;
-        final String[] searchFolders = new String[] { "/sbin/", "/usr/sbin/", "/bin/", "/usr/bin/" };
-
         if (TOOLS.contains(tool)) {
-            ret = true;
+            return true;
         } else {
-            for (String folder : searchFolders) {
-                File fTool = new File(folder + tool);
-                if (fTool.exists()) {
+            for (String folder : DEFAULT_PATH) {
+                File fUnit = new File(folder + tool);
+                if (fUnit.exists()) {
```

Comment:
Are these `Unit` or `Binaries`?

Reply:
Copy-and-paste error. I'll fix it.

Reply:
Addressed in 168582f200f8b7919d89d40c1ea3052aff5f29f0